assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+
+class Person{
+  constructor(attributes){
+  this.name = attributes.name;
+  this.age = attributes.age;
+  this.location = attributes.location
+    }
+
+
+    speak(){
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+}
+
+
+class Instructor extends Person {
+  constructor(attributes) {
+    super(attributes);
+    this.specialty = attributes.specialty;
+    this.favLanguage = attributes.favLanguage;
+    this.catchPhrase = attributes.catchPhrase;
+  }
+
+   demo(subject) {
+    console.log(`Today we are learning about ${subject}`);
+  }
+
+   grade(student, subject) {
+    console.log(`${student.name} receives a perfect score on ${subject}`);
+  }
+}
+
+
+class Student extends Person {
+  constructor(attributes) {
+    super(attributes);
+    this.previousBackground = attributes.previousBackground;
+    this.className = attributes.className;
+    this.favSubjects = attributes.favSubjects;
+
+   }
+  listsSubject() {
+    this.favSubjects.forEach(subject => console.log(subject));
```

Comment:
Excellent use of `forEach` and an arrow function! This is what modern JS looks like.